doc/semitrans.xml

```diff
@@ -12,18 +12,20 @@
 <ManSection>
   <Attr Name = "GeneratorsSmallest" Arg = "S" Label = "for a transformation
     semigroup"/>
-  <Returns>A generating set of transformations.</Returns>
+  <Returns>The smallest generating set of <A>S</A>.</Returns>
   <Description>
-    <C>GeneratorsSmallest</C> returns the lexicographically least collection
-    <C>X</C> of transformations such that <A>S</A> is generated by <C>X</C> and
-    each <C>X[i]</C> is not generated by <C>X[1], X[2], ..., X[i-1]</C>. <P/>
+    For a transformation semigroup <A>S</A>, <C>GeneratorsSmallest</C> returns
+    the   lexicographically least set of elements <C>X</C> such that <C>X</C>
+    generates <A>S</A> as a semigroup. The set <C>X</C> is lexicographically
+    ordered and has the property that each <C>X[i]</C> is not generated by
+    <C>X[1], X[2], ..., X[i-1]</C>. <P/>
 
-    Note that it can be difficult to find this set of generators, and that it
-    might contain a substantial proportion of the elements of the semigroup. <P/>
+    It can be difficult to find the set of generators <C>X</C>, and it might
+    contain a substantial proportion of the elements of <A>S</A>. <P/>
 
     The comparison of two transformation semigroups via the lexicographic
-    comparison of their sets of elements is the same relation as the
-    lexicographic comparison of their <C>GeneratorsSmallest</C>. However, due to
+    comparison of their sorted elements set is the same relation as the
+    lexicographic comparison of their smallest generator sets. However, due to
```

Comment:
That whole sentence is horrible.  You made a valiant attempt to save it, but it wants rewriting.

I might go for:
"Two transformation semigroups have the same set of elements if and only if their smallest generating sets are equal."